index.html

```diff
@@ -260,5 +260,47 @@ <h3>LET’S CONNECT</h3>
 
             <!-- End: About Section-->
 
+
+            <!-- Start: Contact Form-->
+            <section id="contact-section" class="form-holder">
+                <div class="form-wrapper">
+                    <div class="form-description">
+                    <h2>Contact me</h2>
+                    <p>
+                        If you have an application you are interested in developing, a
+                        feature that you need built or a project that needs coding. I'd love
+                        to help with it!
+                    </p>
+                    </div>
+                    <form action="https://formspree.io/f/myyaorpl" method="post">
```

Comment:
- [ ] Great implementation :+1: but I think the form action is not functioning well at the moment as the form could not submit successfully yet. Kindly check your `formspree` id :wink: 

![image](https://github.com/gfrancoarq/portfolio2023/assets/99715640/e1974421-a0e1-4351-9061-2f00263440f1)